template-parts/homepage-landing.php

```diff
@@ -29,6 +29,21 @@
     <!-- If we need scrollbar -->
     <div class="swiper-scrollbar"></div>
 </div>
+<!-- Fetch JSON data for dynamic slider numbers -->
+<script type="text/javascript">
+	jQuery.getJSON("<?php echo get_template_directory_uri(); ?>/data/cdr_compressed.json", function (cdrData) {
+		var cdrTotalRecords = cdrData.meta.num_records;
+		jQuery("#cdrTotal").append(cdrTotalRecords.toLocaleString('en'));
```

Comment:
can we make sure all selectors used for js are prefixed with js- so it's obvious that these selectors should not be used for styles and are attached to js functionality.

Reply:
is the toLocaleString('en') necessary given we don't have any internationalization on this site -- everything's english?

Reply:
Ah, I see -- this is to format with commas. 👍

Reply:
The selectors should be updated now.